### ISSUE TITLE
Questions about the code

### ISSUE THREAD

**Issue by JayZhu0104** [yashkant/sam-textvqa#6]
Hi,
I read your code carefully, but I have questions about some of the code.

- `attention_mask_quadrants: [1,2]`
I don't know what's the meaning of "attention_mask_quadrants"  in .yml files. Do you mean to stop paying attention to the relationship between 1 and 2
- `self.matrix_type_map`
`self.matrix_type_map = {
            "none": "1",
            "share3": "3",
            "share5": "5",
            "share7": "7",
            "share9": "9",
        }`
If I set it to none, does that mean focusing only on the one relationship, or on all, like the traditional transformers

Thanks!

**Comment by yashkant** [comment]
Hi @JayZhu0104,

> `attention_mask_quadrants`

Please see my reply here. -- https://github.com/yashkant/sam-textvqa/issues/2#issuecomment-717323494
Masking [1,2] means we don't let the question-tokens attend to themselves, or the object/ocr tokens in the self-attention layers.

> `self.matrix_type_map`

The values of this dictionary define the value of context (c) of the attention heads, see Fig. 3 and the paragraphs below it for the explanation of what "context" of the head means.

**Comment by JayZhu0104** [comment]
Thanks for your reply!
